src/reactRouter.re

```diff
@@ -35,10 +35,52 @@ module Route = {
     );
 };
 
+module Switch = {
+  [@bs.module "react-router-dom"] external _switch : ReasonReact.reactClass = "Switch";
+  let make = (children) =>
+    ReasonReact.wrapJsForReason(
+      ~reactClass=_switch,
```

Comment:
Nitpick: you can simply name the external `reactClass`, then you can avoid passing the variable

```
[@bs.module "react-router-dom"] external reactClass : ReasonReact.reactClass = "Switch";

ReasonReact.wrapJsForReason(~reactClass, ~props=Js.Obj.empty(), children);
```